src/shared/api.ts

```diff
@@ -277,6 +277,16 @@ export const geminiModels = {
 		inputPrice: 0,
 		outputPrice: 0,
 	},
+
```

Comment:
```suggestion
```

Reply:
suggestion: commit this to remove extraneous line to match formatting in other entries in the same file

Reply:
Thanks for the input @lloydchang 🙏